katka/models.py

```diff
@@ -122,3 +122,15 @@ class SCMRelease(AuditedModel):
     to_hash = models.CharField(max_length=64)
     # points to the pipeline run that created the release, and contains the information on the deployment pipeline
     scm_pipeline_run = models.ForeignKey(SCMPipelineRun, on_delete=models.PROTECT)
+
+
+class ApplicationMetadata(AuditedModel):
+    key = models.CharField(max_length=50)
+    value = models.CharField(max_length=200)
```

Comment:
How did you pick these max lengths?

Reply:
randomly based on the secrets

Reply:
As we don't know what exactly we are going to put there, any number is a guess I think

Reply:
That's true. While we are pre v1 I guess it doesn't matter too much. In my head they might also become json blobs, but we can also always swap to a proper KV database if this becomes necessary